contracts/modules/STO/Capped/CappedSTO.sol

```diff
@@ -42,14 +44,16 @@ contract CappedSTO is CappedSTOStorage, STO, ReentrancyGuard {
      * @param _rate Token units a buyer gets multiplied by 10^18 per wei / base unit of POLY
      * @param _fundRaiseTypes Type of currency used to collect the funds
      * @param _fundsReceiver Ethereum account address to hold the funds
+     * @param _treasuryWallet Ethereum account address to receive unsold tokens
      */
     function configure(
         uint256 _startTime,
         uint256 _endTime,
         uint256 _cap,
         uint256 _rate,
         FundRaiseType[] memory _fundRaiseTypes,
-        address payable _fundsReceiver
+        address payable _fundsReceiver,
+        address _treasuryWallet
```

Comment:
Should we add the preMint boolean here as well so it can be set on initialisation? (same for USDTieredSTO)

Reply:
I don't think so, because if we add the `preMint` boolean here then we will be spending more gas when issuer chooses `mint on buying`. Anyway, the issuer has to call the `allowPreMint()` function to facilitate the pre-mint functionality so it doesn't make too much sense.

Reply:
I was assuming that if `preMint` was set to true, then `_allowPreMint` would be called on initialisation automatically (i.e. it becomes a one-step process, rather than two-step process). You're right that if `preMint` was false it would be slightly more expensive than just having this as a default. Have asked @Mawadda-Poly this question over slack.